core/controllers/reader.py

```diff
@@ -1012,15 +1012,20 @@ def get(self):
 
         skill_ids = self.request.get('skill_ids').split(',')
         question_count = self.request.get('question_count')
+        sort_by_mastery = self.request.get('sort_by_mastery')
 
         if not question_count.isdigit() or int(question_count) <= 0:
             raise self.InvalidInputException(
                 'Question count has to be greater than 0')
 
+        if not (sort_by_mastery == '0' or sort_by_mastery == '1'):
```

Comment:
I agree that true/false is more semantic, but I chose to use 0/1 because we can just use int(sort_by_mastery) to convert the string, instead of an if/else statement. I'm not sure which is better though.

Reply:
Always go with the more semantic version.

Reply:
Done.